README.md

```diff
@@ -0,0 +1,28 @@
+# EPICS IOC for uTCA crate management
+
+[![Continuous Integration
+Status](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/actions/workflows/build.yml/badge.svg)](https://github.com/lnls-dig/ipmi-mgr-epics-ioc/actions)
```

Comment:
As I could understand, this will take the [default branch action status because we are not using `branch` parameter][1].

When we merge this, it won't trigger a workflow after merging, according to our CI rule. Will the badge status still be correct?

[1]: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge#using-the-branch-parameter

Reply:
Hm, I guess that, in this case, as the `branch` parameter isn't being used, the badge will always shows the status of the last triggered CI workflow, so it will not display the status of the merging because it not triggers the workflow, as you notice, but will display the status of the last push or PR.

Anyway, that badge is almost a decoration, and I can remove it if the status could cause any misunderstandings.